src/components/SearchBar/styles.module.css

```diff
@@ -5,4 +5,52 @@
   border-radius: 32px;
   overflow: hidden;
   box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.5);
+}
+
+.innermenu{
+  max-width: 720px;
+  margin: auto;
+  width: 100%;
+  border-radius: 32px;
+  overflow: hidden;
+  box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.5);
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%);
+  border-radius: 20px;
+  background-color: white;
+  overflow: hidden;
+  height: 48px;
+}
+@media (max-width: 720px) {
+  body .container{
+    width: 100%!important;
+  }
+}
+
+.inputContainer {
+  width: calc(100% - 80px);
+  margin: auto 16px;
+}
+
+.searchbar {
+  width: 100%;
+  outline: none;
+  border: none;
+}
+
+.searchButton {
```

Comment:
this messes us the design on the homepage, so i say that for now we just go back to how it was without the colored button, and we can fix this up later